docs/start-deployment.md

```diff
@@ -54,23 +56,85 @@ $ sudo ln -s /usr/local/bin/docker-compose /usr/bin/docker-compose
 docker-compose --version
 ```
 
-## 4. Clone the repository and set it up
+## 4. Clone the repository
 
-```
+We've provided a [Github repository](https://github.com/CitizenLabDotCo/citizenlab-docker) with a minimal premade configuration to run the platform on a single server. 
+
+```bash
 $ sudo apt-get install -y git
 
 $ git clone https://github.com/CitizenLabDotCo/citizenlab-docker.git
 
 $ cd citizenlab-docker
+```
+
+## 5. Provide configuration values
 
+Before we can launch the application, we need to provide configuration values. You can choose some of these values yourself, but for some you will need to create external accounts. All configuration values need to be provided in the `.env` file.
+
+:::note
+You can edit the `.env` file straight on the server using a command line text editor. Nano is a good, easy to user one. Type `nano .env` to open it.
+:::
+
+### Secrets
+
+In order to provide the application with strong internal passwords for the database and user authentication, we have provided a script that generates them for you.
+
+```bash
 $ sh install.sh
+```
+### Admin user
+
+Provide the initial email, password, first and last name of the admin user. You can only set them here initially, once the application has started this no longer has an effect.
+
+### Domain name and DNS
 
-# this step may take a while since it will download all images needed to run the project
-$ docker-compose run --rm web rake db:create db:schema:load
+`CL_SETTINGS_HOST=` controls the domain name of your platform. To start off, you could just set it to the external IP address of your server and access it like that.
+
+To configure the proper domain name, you should change the setting and configure your DNS hosting provider to direct traffer towards the server. The docker-compose setup provides auto-provisioning https, by using Caddy as the webserver, so the connection will always be secure. Adding the A and AAAA DNS records for your domain name, towards your server, is enough to make it work.
```

Comment:
should this read `direct traffic towards the server.` instead of `direct traffer towards the server.`?